tests/unit/core/test_installAddon.js

```diff
@@ -601,6 +601,48 @@ describe(__filename, () => {
         .returns(false);
     });
 
+    describe('isAddonEnabled', () => {
+      it('returns add-on status', async () => {
+        const fakeAddonManager = getFakeAddonManagerWrapper();
+        let guid;
+        const addon = createInternalAddon({
+          ...fakeAddon,
+          _addonManager: fakeAddonManager,
+        });
+        const { root } = renderWithInstallHelpers({
+          ...addon,
+          addon: { guid },
+          _addonManager: fakeAddonManager,
+        });
+        const { isAddonEnabled } = root.instance().props;
+        const isEnabled = await isAddonEnabled();
+
+        expect(isEnabled).toEqual(true);
```

Comment:
is this acceptable/sufficient? or do i need to show it returning false and true individually?

Reply:
that's fine.